puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/release/source/foreman-develop-source-release.yaml

```diff
@@ -1,5 +1,5 @@
 - job:
-    name: foreman-develop-test
+    name: foreman-develop-source-release
     project-type: pipeline
     sandbox: true
     quiet-period: 2700
```

Comment:
We've also noticed this might be a bit long. I wouldn't mind if we lowered this but doesn't have to be now.